src/components/PageElements/Elements/ReviewResponseElement.tsx

```diff
@@ -1,39 +1,41 @@
 import React from 'react'
-import { Icon, Grid } from 'semantic-ui-react'
+import { Icon } from 'semantic-ui-react'
 import { ReviewResponse } from '../../../utils/generated/graphql'
 import { ElementDecisionLabel } from '../../Review'
 
 interface ReviewResponseElementProps {
   isCurrentReview: boolean
   isConsolidation: boolean
+  isDecisionVisible?: boolean
   shouldDim?: boolean
-  shouldHideDecision?: boolean
   reviewResponse: ReviewResponse
 }
 
 const ReviewResponseElement: React.FC<ReviewResponseElementProps> = ({
   isCurrentReview,
   isConsolidation,
-  reviewResponse,
+  isDecisionVisible = true,
   shouldDim = false,
+  reviewResponse,
   children,
-  shouldHideDecision = false,
 }) => {
   const backgroundClass = isCurrentReview ? 'changeable-background' : ''
   const dimClass = shouldDim ? 'dim' : ''
 
+  const canReviewerEdit = (isCurrentReview && !!reviewResponse?.comment) as boolean
+
   return (
     <div className={`response-container ${backgroundClass} ${dimClass}`}>
       <div className="review-response-content ">
-        {!shouldHideDecision && (
-          <ElementDecisionLabel isConsolidation={isConsolidation} reviewResponse={reviewResponse} />
-        )}
-        {!reviewResponse.comment ? null : (
-          <div className="comment-container">
-            <Icon name="comment alternate outline" color="grey" />
-            <p className="secondary">{reviewResponse.comment}</p>
-          </div>
-        )}
+        <ElementDecisionLabel
+          isConsolidation={isConsolidation}
+          reviewResponse={reviewResponse}
+          isDecisionVisible={isDecisionVisible}
+        />
+        <div className="comment-container">
+          {canReviewerEdit ? <Icon name="comment alternate outline" color="grey" /> : null}
```

Comment:
Hmm, the only place I could find in Miro where the comment icon is hidden is for application change request that have not been updated. So i don't quite understand this, bit of logic

Reply:
Yes, that was not correct. I updated now, was just trying to keep the same logic as before (after removing the statement around all) but now revert to only check for reviewResponse.comment is there and show icon: 9d70064